src/transport.rs

```diff
@@ -2,56 +2,71 @@
 
 extern crate hyper;
 
-use self::hyper::buffer::BufReader;
-use self::hyper::header::ContentType;
-use self::hyper::status::StatusCode;
 use self::super::{Error, Result};
-use hyper::Client;
-use hyper::client::Body;
-use hyper::client::response::Response;
+use hyper::Body;
+use hyper::client::{Client, HttpConnector};
+use hyper::{Method, Request, Response, StatusCode};
 use hyper::header;
-use hyper::method::Method;
-use hyper::mime;
-use hyperlocal::DomainUrl;
+use hyper_openssl::HttpsConnector;
+use hyper::rt::{Future, Stream};
+use hyperlocal::Uri as DomainUri;
+use hyperlocal::UnixConnector;
+use mime::Mime;
 use rustc_serialize::json;
 use std::fmt;
-use std::io::Read;
-
-pub fn tar() -> ContentType {
-    ContentType(mime::Mime(
-        mime::TopLevel::Application,
-        mime::SubLevel::Ext(String::from("tar")),
-        vec![],
-    ))
+use std::cell::{RefMut, RefCell};
+use std::io::{BufReader, Cursor};
+use std::io::{Read, Write};
+use tokio::runtime::Runtime;
+
+trait InteractiveStream : Read + Write { }
```

Comment:
I suspect this might be left over from when I first split this PR out from some related shiplift changes, it doesn't look used.